### ISSUE TITLE
Question: Support for photons/keV completed?

### ISSUE THREAD

**Issue by MadFisa** [chianti-atomic/chiantipy#399]
Hi,

I have seen that units for flux can be changed using a .chiantirc files. Is the support for flux in photons and wavelength in keV completed?

**Comment by kdere** [comment]
that was the original intention.  There is some support for having the flux in photon that has worked in the past.  Have not tried it lately.  There is no support for the wavelength in keV.  but T(Kelvin)/constants.boltzmannEv should convert to eV.

**Comment by MadFisa** [comment]
So you would recommend not relying on it (the built-in functions) and instead do it manually?

**Comment by kdere** [comment]
The use of photons should be OK.  I have used it in the past but all of the routine expect a temperature in Kelvin

**Comment by MadFisa** [comment]
What about wavelengths in keV?

**Comment by kdere** [comment]
right now, only eV

**Comment by MadFisa** [comment]
I am sorry, I am confused. 
In .chiantirc files, options for wavelength are in angstrom,nm and keV. 
But looking through the code and as you mentioned above, it looks like wavelength is actually expected in eV

https://github.com/chianti-atomic/ChiantiPy/blob/b92bcd6c5e420043dbcf426689c56af8d1d7c5e5/ChiantiPy/core/Ion.py#L2157-L2158

Isn't the rc file options and code contradictory then?

**Comment by kdere** [comment]
yes, they are contradictory.  I have not looked at the in a long time.  Will try to sort it out.

**Comment by kdere** [comment]
I have looked into this.  Implementing the full range of wavelength possibilities is doable for line spectra but would take some work.  However, it is more complex for the continua so I am not going to try to implement the wavelength possibilities right now.

**Comment by kdere** [comment]
I have decided to go ahead an implement the use of units other than angstroms for spectral line calculations.  These include nm and energies in eV and keV.  However, they will not work in spectral calculations and the user will be warned.

**Comment by MadFisa** [comment]
I see that it has been implemented in the new release, thank you for your efforts. I think it is better to leave the issue open as the feature has not been implemented fully (since continuum calculation still not work), for future reference.
